### PR TITLE
MOD: not only support json get from bundle,but also support image res…

### PR DESCRIPTION
**Pull request by upworldcjw**
support image get from bundle

### CODE REVIEW

File: lottie-ios/Classes/AnimatableLayers/LOTLayerView.m [hunk 1]
```diff
@@ -319,7 +320,15 @@ - (NSString*)description {
...
   if (_layerModel.imageAsset.imageName) {
     NSArray *components = [_layerModel.imageAsset.imageName componentsSeparatedByString:@"."];
-    UIImage *image = [UIImage imageNamed:components.firstObject];
+    UIImage *image = nil;
+    if ([LOTContext getRegisterBundle] == [NSBundle mainBundle]) {
+      image = [UIImage imageNamed:components.firstObject];
+    }else{
+      NSBundle *bundle = [LOTContext getRegisterBundle];
+      NSString *bundleName = [[bundle bundlePath] lastPathComponent];
+      image = [UIImage imageNamed:[bundleName stringByAppendingPathComponent:components.firstObject]];
```
Comment (fnazrala) [inline lottie-ios/Classes/AnimatableLayers/LOTLayerView.m]: You can use this 
```UIImage *image = [UIImage imageNamed:components.firstObject inBundle:bundle compatibleWithTraitCollection:nil];```

**Comment by fnazrala** [comment]
You must do the same here. https://github.com/airbnb/lottie-ios/pull/210/files#diff-616358630013b78fb923571fd7ed6314R345

File: lottie-ios/Classes/AnimatableLayers/LOTLayerView.m (hunk 1 above)
Comment (JulienLacr0ix) [inline lottie-ios/Classes/AnimatableLayers/LOTLayerView.m]: Can you merge this PR please ?
I can't retrieve image in custom bundle.

Thx a lot ! ;-)

Comment (fnazrala) [inline lottie-ios/Classes/AnimatableLayers/LOTLayerView.m]: I'm not from lottie's team, i need this change too.

**Comment by buba447** [comment]
I think we are going to go with the solution found here https://github.com/airbnb/lottie-ios/pull/238 
Thanks for the work!

**Pull request closed without merging**